### PR TITLE
Avoid unsafe characters for subvolumes path

### PR DESCRIPTION
**Pull request by joseivanlopez**
PBI: https://trello.com/c/m7eAH7Dw/525-storage-subvolumes

### CODE REVIEW

File: src/lib/y2partitioner/dialogs/btrfs_subvolume.rb [hunk 1]
```diff
@@ -140,44 +151,50 @@ def focus
...
-        # @return [Boolean] true if path does not exist
-        def uniqueness_validation
-          return true unless exist_path?
+        # @return [String, nil] nil if the path does not contain illegal characters
+        def content_error
+          regexp = Regexp.new(".*[#{ILLEGAL_CHARS}]+.*")
 
-          Yast::Popup.Error(format(_("Subvolume name %s already exists."), value))
-          false
+          return nil unless regexp.match?(value)
```
Comment (jreidinger) [inline src/lib/y2partitioner/dialogs/btrfs_subvolume.rb]: this is not needed to construct such complex regexp. Simply use
`return nil unless regexp.match?(/[#{ILLEGAL_CHARS}]/)`

File: src/lib/y2partitioner/dialogs/btrfs_subvolume.rb [hunk 2]
```diff
@@ -140,44 +151,50 @@ def focus
...
+
+          _("Subvolume path contains illegal characters. Be sure it does\n" \
+            "not include white spaces, tabs, line breaks or commas.")
```
Comment (jreidinger) [inline src/lib/y2partitioner/dialogs/btrfs_subvolume.rb]: hmm, this is not much robust when we extend list of illegal characters. Why not simply list them out? with maybe some escaping with inspect?

Comment (lslezak) [reply src/lib/y2partitioner/dialogs/btrfs_subvolume.rb]: Hm, displaying something like `\t` would be really confusing for the beginners as you have `Tab` written on the physical keyboard. Similar to `\n` and others, I'd definitely avoid that.

Comment (joseivanlopez) [reply src/lib/y2partitioner/dialogs/btrfs_subvolume.rb]: As suggested by @lsleza, I have extended the message with "or similar special characters".

**Pull request merged**